docs/source/guides/quickstart.md

```diff
@@ -1,48 +1,195 @@
 # NuCypher Quickstart
 
-## Ursula
 
-### Install NuCypher
+## A Note about Side Channels
+
+The NuCypher network does not store or handle an application's data; instead - it manages access *to* application data.
+Management of encrypted secrets and public keys tends to be highly domain-specific - The surrounding architecture
+will vary greatly depending on the throughput, sensitivity, and sharing cadence of application secrets.
+In all cases, NuCypher must be integrated with a storage and transport layer in order to function properly.
+Along with the transport of ciphertexts, a nucypher application also needs to include channels for Alice and Bob 
+to discover each other's public keys, and provide policy encrypting information to Bob and Enrico.
+ 
+##### Side Channel Data:
+  - Secrets:
+    - Message Kits (Ciphertext)
+  - Identities:
+    - Alice Verifying Key
+    - Bob Encrypting Key
+    - Bob Verifying Key
+  - Policies:
+    - Policy Encrypting Key 
+    - Labels
+
+## Running an Ethereum Node
+
+Operation of a decentralized NuCypher character [`Alice`, `Bob`, `Ursula`] requires
+a connection to an Ethereum node and wallet.
+
+To run a Goerli-connected Geth node in *fast* syncing mode:
 
 ```bash
-$ pip3 install -U nucypher
+$ geth --goerl
```

Comment:
You, of course, meant `--goerli`, not `goerl`?

Reply:
Yep! Looks like an accidental.  Note that this PR is WIP and I have not fully responded to outstanding RFCs.

Reply:
:heavy_check_mark: